app/src/lib/git/rebase.ts

```diff
@@ -87,6 +87,24 @@ export async function getRebaseContext(
   return null
 }
 
+/**
+ * Attempt to read the `.git/REBASE_HEAD` file inside a repository to confirm
+ * the rebase is still active.
+ */
+async function readRebaseHead(repository: Repository): Promise<string | null> {
+  try {
+    const rebaseHead = Path.join(repository.path, '.git', 'REBASE_HEAD')
+    const rebaseCurrentCommitOutput = await FSE.readFile(rebaseHead, 'utf8')
+    return rebaseCurrentCommitOutput.trim()
+  } catch (err) {
+    log.warn(
+      '[rebase] a problem was encountered reading .git/REBASE_HEAD, so it is unsafe to continue rebasing',
```

Comment:
This adds better error handling to a scenario that @tierninho identified during QA - if we're trying to continue when there no active rebase, we shouldn't try to proceed.